### ISSUE TITLE
Disable NVDA update and quit dialogs

### ISSUE THREAD

**Issue by cmorten** [guidepup/setup#17]
The patched NVDA is launching update dialog boxes which can impact on tests, this should be disabled.

It also will be showing the default dialog for when you quit with `Insert-Q`, we probably want to disable this by default.

See https://github.com/guidepup/guidepup/actions/runs/4030951464/jobs/6929949861 for an example

**Comment by cmorten** [comment]
Fixed by https://github.com/guidepup/setup/commit/285d669f57b0e7d681389a17ef0d1b1c0d2ea1e6
